python/taichi/aot/conventions/gfxruntime140/dr.py

```diff
@@ -60,14 +60,12 @@ def __init__(self, j: Dict[str, Any]) -> None:
         arg_attribs_vec_ = j["arg_attribs_vec_"]
         args_bytes_ = j["args_bytes_"]
         arr_access = j["arr_access"]
-        extra_args_bytes_ = j["extra_args_bytes_"]
```

Comment:
IIUC we won't be able to modify these 140 files since they are for older version? cc: @PENGUINLIONG

Reply:
yes just keep them unused dont remove them

Reply:
or if it really breaks just make it another convension like gfxruntime170 but you then need to do all the parsing again and find a way to upgrade those old modules